public/app/features/alerting/unified/RedirectToRuleViewer.test.tsx

```diff
@@ -76,6 +76,9 @@ describe('Redirect to Rule viewer', () => {
   });
 
   it('should properly decode rule name', () => {
+    // TODO: Fix console warning that happens once CompatRouter is wrapped around this component render
+    jest.spyOn(console, 'warn').mockImplementation();
```

Comment:
From what I can tell, this was actually already `console.warn`'ing inb the browser, when I recreated this test case. So I think our previous mocks were stubbing this out - @konrad147 you seemed to have worked on this area, do you have any thoughts?